urlscan/urlscan.py

```diff
@@ -412,15 +420,15 @@ def extracturls(mesg):
     # lines with more than one entry or one entry that's
     # a URL are the only lines containing URLs.
 
-    linechunks = [parse_text_urls(l) for l in lines]
+    linechunks = [parse_text_urls(l, regex=regex) for l in lines]
 
     return extract_with_context(linechunks,
                                 lambda chunk: len(chunk) > 1 or
                                 (len(chunk) == 1 and chunk[0].url is not None),
                                 1, 1)
 
 
-def extracthtmlurls(mesg):
+def extracthtmlurls(mesg, regex=None):
```

Comment:
regex doesn't get used in this method. Should it be or can it be removed?

Reply:
I apologize, I must have been operating on low caffeine. It is solved.